apps/etterlatte-saksbehandling-ui/client/src/components/behandling/beregningsgrunnlag/InstitusjonsoppholdBP.tsx

```diff
@@ -74,11 +74,12 @@ const Institusjonsopphold = (props: InstitusjonsoppholdProps) => {
             status={null}
           >
             <p>
-              Barnepensjonen skal reduseres under opphold i en institusjon med fri kost og losji under statlig ansvar
-              eller tilsvarende institusjon i utlandet. Regelen gjelder ikke ved opphold i somatiske sykehusavdelinger.
-              Oppholdet må vare i tre måneder i tillegg til innleggelsesmåneden for at barnepensjonen skal bli redusert.
-              Dersom barnet har faste og nødvendige utgifter til bolig, kan arbeids- og velferdsetaten bestemme at
-              barnepensjonen ikke skal reduseres eller reduseres mindre enn hovedregelen sier.
+              Bah Barnepensjonen skal reduseres under opphold i en institusjon med fri kost og losji under statlig
```

Comment:
Egne tekster her?

Reply:
Det er egen text for OMS i InstitusjonsoppholdOMS mulig det gjenstår noe tekstarbeid. Skal ta en gjennomgang med Mari i Dev